Sources/SwiftProtobuf/TextFormatScanner.swift

```diff
@@ -1157,6 +1180,9 @@ internal struct TextFormatScanner {
                         break
                     }
                     p += 1
+                    if p - start > maxFieldNumLength {
```

Comment:
In this case, we could also directly check fieldNum against the value instead of doing something length based, not sure if you'd rather that.

Reply:
As above.